### PR TITLE
feat: fixing the color picker for maps

### LINKED ISSUE #1573: Maps: Fix hex code error on customization options
Maps customization options is not working, because the color picker/dropdown can't find the default hex code. Other graph types have a default hex code, we need to add a default, and we need to add hex codes to all examples of maps.

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #1573

### CODE REVIEW

**Comment by Palakp41** [comment]
@zoesteinkamp Can you update `Maps/view.tsx` file to include the `DEFAULT_THRESHOLDS_GEO_COLORS` constant just so that it' is consistent everywhere?

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1646)

**Review by Palakp41 — APPROVED** [review]

File: src/visualization/types/Map/view.tsx [hunk 1]
```diff
@@ -120,11 +120,7 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         type: 'pointMap',
         colorDimension: {label: 'Value'},
         colorField: '_value',
-        colors: [
-          {type: 'min', hex: InfluxColors.Star, id: '0'},
-          {value: 50, hex: InfluxColors.Star, id: '1'},
-          {type: 'max', hex: InfluxColors.Star, id: '2'},
-        ],
+        colors: DEFAULT_THRESHOLDS_GEO_COLORS,
```
Comment (Palakp41) [inline src/visualization/types/Map/view.tsx:123]: I think you need to update line 130 to use the `DEFAULT_THRESHOLD_GEO_COLORS`.

Comment (zoesteinkamp) [reply src/visualization/types/Map/view.tsx]: did it!

**Pull request merged**